templates/project/programs/_programName_/tests/basic.nix

```
@@ -12,8 +12,8 @@
       {
         imports = [
           sources.modules.ngipkgs
-          sources.modules.services._programName_
-          sources.examples._ProjectName_._programName_
+          sources.modules.programs._programName_
+          sources.examples._ProjectName_._testName_ # i.e _ProjectName_.basic
```

This should be the name for the example, not the test. We also need to modify the one in services

```suggestion
          sources.examples._ProjectName_._exampleName_ # i.e _ProjectName_.basic
```

---

ooh, yess! Done